### ISSUE TITLE
[fixed in TabMix] Conflict with TabMix Plus

### ISSUE THREAD

**Issue by desirepl** [tabkit/tabkit2#104]
This conflict has a long story. This conflict was from the beginning when this two extensions was available.
What happens: when tabs should be shown along the left edge and both of extensions are on then after starting firefox all tabs aligned at the top of the created by Tab Kit 2nd Edition left frame.
When we going to settings of Tab Kit, select "at the top", OK, then again "along the left" and OK - then it fixes for current session. When ff is restarted - then all tabs again aligned at the top of the left frame.
I think it's about who's loads first/last. When Tab Mix loads/started to work after Tab Kit then it takes control over tabs alignment and corrupting tabs alignment.

**Comment by desirepl** [comment]
Can't check this issue with latest test version of TabKit 2nd Edition, because author of TabMix fixed this at his side by blocking of tab control functions when TabKit 2nd Edition installed and if vertical tabs selected after my request.
So maybe anyone can check who didn't upgraged TabMix?

**Comment by PikachuEXE** [comment]
Thanks for the update!

**Comment by desirepl** [comment]
Problem raised again...
If i correct, yesterday TabKit was updated and today after restarting FF this issue exists again.
Did they changed something that TabMix can't find previous names/variables and thinks that there is no TabKit or vertical alignment of tabs?..

**Comment by PikachuEXE** [comment]
Yes it's trying to detect `tabkitGlobal` which is useless stuff that I removed in latest version to remove usage of `eval`
The TaMix Plus code:
https://bitbucket.org/onemen/tabmixplus/src/3d713f877bc78f446c7f648fe08c95848378c9a2/chrome/content/extensions/extensions.js?at=default&fileviewer=file-view-default#extensions.js-469
I think it should try to detect `tabkit` in `window` which shouldn't be changed for a long time.

**Comment by PikachuEXE** [comment]
Reported in
https://bitbucket.org/onemen/tabmixplus/issues/22/compatibility-with-tab-kit
